dev/build/windows/makecoq_mingw.sh

```diff
@@ -1816,7 +1816,7 @@ function make_addon_coquelicot {
   installer_addon_dependency_end
   if build_prep_overlay coquelicot; then
     installer_addon_section coquelicot "Coquelicot" "Coq library for real analysis" ""
-    logn autogen ./autogen.sh
+    logn autoreconf -i -s
```

Comment:
I am not sure this does what is intended. Please note that "logn" expects a logging name as first argument and the rest are the commands. You probably meant log1 autoreconf which uses the first argument as logging name and command.

Reply:
Oops, thanks!